src/App.vue

```diff
@@ -281,9 +296,11 @@
             },
             layoutReadyEvent: function(newLayout){
                 console.log("Ready layout: ", newLayout)
+                this.layout = newLayout
```

Comment:
can pr "support vue3" only for upgrading vue from 2 to 3?  bug fix or implementation change move to another pr.

Reply:
all changes based on https://github.com/jbaysolutions/vue-grid-layout/compare/master...vue3-webpack

this is what origin author do to support vue3